taxi/views.py

```diff
@@ -1,47 +1,52 @@
 from django.shortcuts import render
 from django.views import generic
-
+from django.contrib.auth.decorators import login_required
+from django.contrib.auth.mixins import LoginRequiredMixin
```

Comment:
The `LoginRequiredMixin` is appropriately used with class-based views to ensure that only authenticated users can access these views. This is a good practice for securing views.